### ISSUE TITLE
Typo in Configuring WIA for chrome command block

### ISSUE THREAD

**Issue by CaiusAjiz** [MicrosoftDocs/windowsserverdocs#3208]
Current command is:
Set-AdfsProperties -WIASupportedUserAgents ((Get-ADFSProperties | Select -ExpandProperty WIASupportedUserAgents) + "Mozilla/5.0 (Windows NT")

And has a typo at the end which leads to the user agent to go into ADFS incorrectly. Goes in as 'Mozilla/5.0 (Windows NT' & 'Mozilla/5.0 (Macintosh; Intel Mac OS X'

Command should be:
Set-AdfsProperties -WIASupportedUserAgents ((Get-ADFSProperties | Select -ExpandProperty WIASupportedUserAgents) + "Mozilla/5.0 (Windows NT)"

Same problem for the Mac one where the quote is before the closing bracket.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3d6a8f34-a392-d1f2-fb61-0d34bca5ab0d
* Version Independent ID: a6cb49c5-2ac2-77e8-7f39-493713f7e9a8
* Content: [Configuring intranet forms-based authentication for devices that do not support WIA](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/operations/configure-intranet-forms-based-authentication-for-devices-that-do-not-support-wia#feedback)
* Content Source: [WindowsServerDocs/identity/ad-fs/operations/Configure-intranet-forms-based-authentication-for-devices-that-do-not-support-WIA.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/identity/ad-fs/operations/Configure-intranet-forms-based-authentication-for-devices-that-do-not-support-WIA.md)
* Product: **windows-server-threshold**
* Technology: **identity-adfs**
* GitHub Login: @billmath
* Microsoft Alias: **billmath**

**Comment by jonimattila** [comment]
Yes, the second parenthesis is missed by intent – we need substring function used by ADFS to match any Windows version on NT kernel (Windows 2000 and later).

**Comment by illfated** [comment]
(Ref. #4342)

**Comment by IngridAtMicrosoft** [comment]
Thanks for your dedication to our documentation. If we have been unable to review your issue in a timely manner, we sincerely apologize for the delayed response. We have captured the informtion you have provided for review. We are closing this issue for now, but if you feel that it's still a concern, please respond and let us know. If you determine another possible update to our documentation, please don't hesitate to reach out again. #please-close

**Comment by IngridAtMicrosoft** [comment]
#assign:IngridAtMicrosoft 
gh issue comment 3190 -b #assign:IngridAtMicrosoft
